src/components/AppDialog/AppDialog.component.tsx

```diff
@@ -0,0 +1,58 @@
+import { IconButton, Typography } from '@mui/material';
+import Dialog from '@mui/material/Dialog';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import React from 'react';
+import crossIcon from '../../assets/icons/cross.svg';
+import { AppDialogProps } from './AppDialog.types';
+
+export default function AppDialog({
```

Comment:
We decided not to use default exports, it can sometimes become unclear when searching the project for references.

Please change to arrow function